app/models/order_item.rb

```diff
@@ -0,0 +1,11 @@
+class OrderItem < ApplicationRecord
+  belongs_to :order
+  belongs_to :item
+
+  def create(item_id, quantity)
```

Comment:
looks like this might be some old code that was left in. try to make sure you are deleting code you aren't using. that would also bring up your test coverage!